tests/unit/lms/services/auto_grading_test.py

```diff
@@ -39,8 +39,8 @@ def test_get_last_sync(self, svc, db_session, assignment):
     def test_create_grade_sync(self, svc, db_session, assignment):
         creator = factories.LMSUser()
         lms_users = factories.LMSUser.create_batch(5)
-        grades = {lms_user: random.random() for lms_user in lms_users}
         db_session.flush()
+        grades = {lms_user: random.random() for lms_user in lms_users}
```

Comment:
Another one I don't know why it didn't fail before, without the flush these don't have a SQLA identity in the dict.